public/index.html

```diff
@@ -133,55 +130,204 @@ <h3 class="restaurant-name"> Succulent Society Cafe </h3>
 
     <!-- Display 2 -->
     <div id="display2">
-      <!-- Fixed Restaurant Banner -->
-      <div class="restaurant-banner">
-        <div class="image-holder">
-          <img src="/images/restaurant_banner.jpg" alt="Logo" class="banner-image">
-        </div>
-        <div class="banner-body">
-          <div>
-            <h3 class="restaurant-name"> Succulent Society Cafe </h3>
-            <p>Canadian, Burgers - 4.6<i class="fa-solid fa-star"></i> - 4.6km away - $$$ - (604) 123-4567</p>
+      <section class="website-body">
+        <!-- Fixed Restaurant Banner -->
+        <div class="restaurant-banner">
+          <div class="image-holder">
+            <img src="/images/restaurant_banner.jpg" alt="Logo" class="banner-image">
+          </div>
+          <div class="banner-body">
+            <div>
+              <h3 class="restaurant-name"> Succulent Society Cafe </h3>
+              <p>Canadian, Burgers - 4.6<i class="fa-solid fa-star"></i> - 4.6km away - $$$ - (604) 123-4567</p>
+            </div>
+            <label class="delivery">
+              <input type="checkbox" name="color_mode" id="toggle" value="1">
+              <label data-on="Delivery" data-off="Pickup" class="pickup"></label>
+            </label>
           </div>
-          <label class="delivery">
-            <input type="checkbox" name="color_mode" id="toggle" value="1">
-            <label data-on="Delivery" data-off="Pickup" class="pickup"></label>
-          </label>
         </div>
-      </div>
-
-      <div class="account-details">
-        <h2>Account Details</h2>
-      </div>
-      <div class="shipping-details">
-        <h2>Shipping Details</h2>
-      </div>
-      <div class="payment-details">
-        <h2>Payment Details</h2>
-      </div>
 
+        <div class="account-details">
+          <h2>Account Details</h2>
+        </div>
+        <div class="shipping-details">
+          <h2>Shipping Details</h2>
+        </div>
+        <div class="payment-details">
+          <h2>Payment Details</h2>
+        </div>
+      </section>
       <!-- Checkout Summary -->
       <section class="order-section">
-        <!-- edit from here -->
         <div class="checkout-cart-container">
-          <div class="checkout-cart-header" id="order-summary-right">
```

Comment:
wasn't sure what this was here for so cleaned it up for now until we review together.